src/jit/inlinepolicy.h

```diff
@@ -216,13 +216,45 @@ class DiscretionaryPolicy : public LegacyPolicy
     const char* GetName() const override { return "DiscretionaryPolicy"; }
 
 private:
+    
+    void ComputeOpcodeBin(OPCODE opcode);
+    enum { MAX_ARGS = 4 };
```

Comment:
Yes. Most inline candidates don't have very many arguments.... with current inliner, we see inline candidate data like:

| #args | percent |
|-------|---------|
| 0     | 57.6    |
| 1     | 26.0    |
| 2     | 10.4    |
| 3     |  3.7    |
| 4+    |  2.3    |